telemetry/definitions/commonDefinitions.json

```
@@ -248,10 +248,25 @@
             "type": "string",
             "description": "Any API-specific errors"
         },
+        {
+            "name": "codeTransformApiName",
```

If we are really going to start counting service calls (that really, really needs careful de-duplication treatment on the clients), then there is no reason for this field to be codetransform-specific.

We already have a "serviceType" field https://github.com/aws/aws-toolkit-common/blob/3b77ad1d8bd07708c0e4dc16c895f09cced41b0d/telemetry/definitions/commonDefinitions.json#L1546 

, so it's more discoverable to name this something like `serviceCall` or `serviceApiName`
```suggestion
            "name": "serviceApiName",
```

---

Thinking more about this - it's true that emitting metrics for every service call has not been particularly helpful, and we do not evaluate those metrics. I'll remove the new changes related to `codeTransform_invokeApi`, but keep `codeTransform_logApi*` as deprecated.